daemon/api.go

```diff
@@ -967,6 +976,119 @@ func deleteCapability(c *Command, r *http.Request) Response {
 	}
 }
 
+// skillGrant is a part of response from getSkills().
```

Comment:
`// skillGrant holds the identification of a slot that has been granted to a skill.`